mephisto/core/operator.py

```diff
@@ -279,6 +279,7 @@ def parse_and_launch_run(
             architect=architect,
             job=job,
         )
+        task_run.assignments_generator_done = False
```

Comment:
Nit - this should also use the setter `update_completion_progress` rather than a direct access, as we are trying to decouple the two and the less the operator needs to touch directly, the better.